### PR TITLE
Added id parameter to bug description files (partially addresses #165)

### LINKED ISSUE #163: Use bugzoo/fork-url param for all bugs
Currently if the `bugzoo/fork-url` param (in *.bug description) is not set `build-bugzoo.py` would look at the name of the bug's folder and use that as a lookup in a hardcoded dict. Problem arise if any bug is moved into another folder like e.g. one with the git-hash's name. Then, the lookup fails and currently also the program terminates.
Creating the `fork-url` out of the  
`"https://github.com/robust-rosin/"+desc["time-machine"]["ros_pkgs"]` would also work, however this would be very specific to the robust project and only guessing.

Thus, to assure that the  `build-bugzoo.py` is a) more flexible and b) more stable one we decided in today's telco to use the bugzoo/fork-url param for all bugs.
This requires:
- [ ] update the readme related to `build-bugzoo.py`
- [ ] update the `build-bugzoo.py` (remove the lookup)
- add this param to all bug descriptions that have been created so far! (I would suggest this can be done for every repo by the person who is working on that repo)
    - [ ] care-o-bot 
    - [ ] geometry2
    - [ ] motoman
    - [ ] ros_comm
    - [ ] universal_robot
    - [ ] kobuki
    - [ ] mavros
    - [ ] turtlebot

Example of an error (here I would need the repo cob_control):
```bash
$ python build-bugzoo.py 
generating manifest for bug [care-o-bot:description] at file (/...../robust/care-o-bot/description.bug)
Traceback (most recent call last):
  File "../../scripts/build-bugzoo.py", line 136, in <module>
    main()
  File "../../scripts/build-bugzoo.py", line 74, in main
    })[package]
KeyError: 'care-o-bot'
```
@ChrisTimperley , @gavanderhoorn this could also be an option which would save us the work to add a param to every `*.bug`. What do you think?
```python
# new param: fork_base_url
robust_base_url =  desc["bugzoo"]["fork_base_url"]  or "https://github.com/robust-rosin/"
url_fork = robust_base_url + desc["time-machine"]["ros_pkgs"]
```

### PR DESCRIPTION
**Pull request by ChrisTimperley**

### CODE REVIEW

File: care-o-bot/105dc16/description.bug [hunk 1]
```diff
@@ -1,5 +1,6 @@
 %YAML 1.1
 ---
+id: description
```
Comment (gavanderhoorn) [inline care-o-bot/105dc16/description.bug]: This doesn't look like a valid value for this field?

**Comment by gavanderhoorn** [comment]
Does this address #163 or #165?

**Comment by ChrisTimperley** [comment]
> Does this address #163 or #165?

Right now, it partially addresses #165 by adding an `id` property to each `.bug` file. #163 and #165 are quite closely related, however, so I may end up addressing them both in this issue.

**Comment by ChrisTimperley** [comment]
Since #163 involves its own set of wholesale changes to the `.bug` files, I'll create a separate PR for that. To resolve #165, we'll need to impose a consistent directory structure. Keeping the `.bug` file in the same directory as its associated artefacts, as suggested by @gavanderhoorn, seems the most robust.

**Comment by ChrisTimperley** [comment]
> Since #163 involves its own set of wholesale changes to the `.bug` files, I'll create a separate PR for that. To resolve #165, we'll need to impose a consistent directory structure. Keeping the `.bug` file in the same directory as its associated artefacts, as suggested by @gavanderhoorn, seems the most robust.

Implementing a consistent directory structure is going to involve a lot of changes, so I'll create a separate PR for that.

**Comment by gavanderhoorn** [comment]
I might be annoying, but I'd rather we stay away from as many assumptions in the code as possible.

Anything that is derived from file or directory names, hashes that need to point to certain repository states and other such things introduce points where the tools can break if such (implicit) assumptions are invalidated for whatever reason.

If possible, making things explicit should be preferred -- even if that increases the cost of maintenance slightly.

**Pull request merged**